src/main/java/com/SAS/LeagueManagement/LeagueManagementController.java

```diff
@@ -0,0 +1,226 @@
+package com.SAS.LeagueManagement;
+
+import com.SAS.League.*;
+import sun.awt.image.ImageWatched;
+
+import java.util.LinkedList;
+
+public class LeagueManagementController {
+
+    private LinkedList<LeagueRankPolicy> rankPolicies;
+    private LinkedList<PointsPolicy> pointsPolicies;
+    private LinkedList<GamesPolicy> gamesPolicies;
+
+    /**
+     * Constructor
+     */
+    public LeagueManagementController() {
+        this.rankPolicies = new LinkedList<>();
+        this.pointsPolicies = new LinkedList<>();
+        this.gamesPolicies = new LinkedList<>();
+        initPolicies();
+    }
+
+    /**
+     * The functions initialize the available policies in our system
+     */
+    private void initPolicies() {
+        //add rank policies
+        rankPolicies.add(new NumberOfWins());
+        rankPolicies.add(new GoalDifference());
+
+        //add points policies
+        pointsPolicies.add(new OnePointForWinAndNoneForDraw());
+        pointsPolicies.add(new TwoForWinOneForDraw());
+        pointsPolicies.add(new ThreeForWinOneForDrawPolicy());
+
+        //add games policies
+        gamesPolicies.add(new OneRoundLeague());
+        gamesPolicies.add(new TwoRoundsLeague());
+        gamesPolicies.add(new ThreeRoundsLeague());
+    }
+
+    /**
+     * The function returns all the available rank policies in our system
+     * @return
+     */
+    public String showRankPolicies(){
+        int counter = 1;
```

Comment:
Maybe you can call a function that receives a list and prints the available options - and in each function of show policies will call it with relevant list.

Reply:
concider doing it in feauture task, right now its ok